docs/UserGuide.md

```diff
@@ -4,53 +4,94 @@
-   2. Select the installation package based on your Operating System. For Apple users select macOS, for Windows users select Windows.
-   3. Follow the instruction guide to install Java on your device. For more information use this guide: https://docs.oracle.com/en/java/javase/23/install/overview-jdk-installation.html
-
-2. Download the latest version of WedLinker from [here](https://github.com/AY2425S1-CS2103T-F15-4/tp/releases/). Select `WedLinker.jar` to begin the download.
-![release.png](images/release.png)
-
-3. Copy the file to the folder you want to use as the _home folder_ for your WedLinker.
-
-4. Open WedLinker by double clicking the program file, `WedLinker.jar`.<br>
-   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
-
-5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+
+   1.1. Open your operating system's terminal. <br>Follow instructions for your operating system if you are unsure how to:
+      <details>
+        <summary><strong>Windows:</strong></summary>
+        <ol>
+          <li>Hold down Windows button and R (⊞ Win + R)</li>
+          <li>Type <code>cmd</code> and press enter</li>
+        </ol>
+      </details>
+      <details>
+        <summary><strong>MacOS:</strong></summary>
+        <ul>
+          <li>Click the Launchpad icon in the Dock, type <code>Terminal</code> in the search field, then click Terminal, OR</li>
+          <li>In the Finder, open the <code>/Applications/Utilities</code> folder, then double-click Terminal.</li>
+        </ul>
+      </details>
+    <br>
+
+   1.2. Type <code>java -version</code> and press enter.
+
+   1.3. If you have `Java 17` installed, the terminal should look like:
+      ```
+        C:\Users\UserName>java -version
+        java version "17.0.12" 2024-07-16 LTS
+        Java(TM) SE Runtime Environment (build 17.0.12+8-LTS-286)
+        Java HotSpot(TM) 64-Bit Server VM (build 17.0.12+8-LTS-286, mixed mode, sharing)
+      ```
+
+<br>
+
+2. If you have `Java 17` installed, proceed to Step 3. If not, follow these instructions:
+
+   2.1. Download `Java 17` from [here](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html)
+   
+    2.2. Select the installation package based on your Operating System.
+   
+    2.3. Follow the instruction guide to install Java on your device. For more information, click [here](https://docs.oracle.com/en/java/javase/23/install/overview-jdk-installation.html)
+
+<br>
+
+3. Download the latest version of WedLinker from [here](https://github.com/AY2425S1-CS2103T-F15-4/tp/releases/). Select `WedLinker.jar` to begin the download.
+   <img alt="release.png" src="images/release.png" width="700" onclick="openModal(this)"/>
+
+<br>
+
+4. Copy the file to the folder you want to use as the _home folder_ for your WedLinker.
```

Comment:
This may sound silly... but I dont know if Dr Wee will pick on this but I think if we assume the users are *really* non-technical, they may not know how to "copy the file" directly after step 3. I feel like to make it clearer maybe could say something like "Once the download is completed, the file will appear in the "Downloads" folder of your computer. Copy the file..."